src/main/kotlin/no/nav/sokos/oppdrag/attestasjon/service/AttestasjonService.kt

```diff
@@ -99,9 +100,20 @@ class AttestasjonService(
                         }
                     }
                     .map { it.copy(hasWriteAccess = hasSaksbehandlerWriteAccess(it, saksbehandler)) }
+                    .sortedWith(sortOppdrag(sortKey))
```

Comment:
sortWith bør skjer etter linje 88

val oppdragListe = temp.filter { hasSaksbehandlerReadAccess(it, saksbehandler) }

fordi når du kommer til oppdragList.subList, så blir det allerede tatt ut den delen som skal vise i paginering